libreconv.gemspec

```
@@ -23,6 +23,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_development_dependency 'bundler', '>= 0'
+  spec.add_development_dependency 'mixlib-shellout'
```

In this case is not a development dependency ex.: https://github.com/ScreenLoop/hubspot-rb/blob/master/hubspot-api-client.gemspec
```suggestion
  spec.add_runtime_dependency 'mixlib-shellout'
```